index.html

```diff
@@ -65,12 +65,15 @@
 ## Modal verbs terminology
 In the present document "shall", "shall not", "should", "should not", "may", "need not", "will", "will not", "can" and "cannot" are to be interpreted as described in clause 3.2 of the ETSI Drafting Rules (Verbal forms for the expression of provisions).
 
+## Definition of mnemonics and syntax function
+In the present documents the mnemonics, the syntax functions, and the syntax descriptors are to be interpreted as described in ([[MPEG-2 TS]]). The **uimsbf** and **bslbf** mnemonics are defined in Section 2.2.6 of ([[MPEG-2 TS]]). The **nextbits()** function is interpreted as in ([[MPEG-2 TS]]).
```

Comment:
There is a typo in "the present documents", no 's'.

Reply:
Fix in f34d1770af58b39e976942d8b1f2152d943c5a76